packages/docs-app/src/styles/_sections.scss

```diff
@@ -35,7 +35,7 @@
 
 // CSS examples
 
-#{reference("pt-file-upload")},
+#{reference("pt-file-input")},
```

Comment:
Is this one missed from https://github.com/palantir/blueprint/pull/1880 ?

Reply:
Looks like it. Good thing we have a script!